bin/npg_publish_bionano_run.pl

```diff
@@ -83,16 +92,29 @@ sub run {
                 $collection, q[']);
     my $wh_schema = WTSI::DNAP::Warehouse::Schema->connect;
     foreach my $dir (@dirs) {
-        my %args = (
+        my @init_args = (
             directory   => $dir,
             mlwh_schema => $wh_schema,
             irods       => $irods,
         );
         if (defined $output_dir) {
-            $args{'output_dir'} = $output_dir;
+            push @init_args, output_dir => $output_dir;
+        }
+        if ($enable_rmq) {
+            push @init_args, enable_rmq => 1;
+            if (defined $channel) {
```

Comment:
Maybe is worth to show a warning if any of these options (channel, exchange, rounting_key_prefix) are set but enable_rmq is not set?